package.json

```diff
@@ -64,6 +64,8 @@
     "build": "scripts/use_node scripts/build --all-platforms",
     "start": "scripts/use_node scripts/opensearch_dashboards --dev",
     "start:docker": "scripts/use_node scripts/opensearch_dashboards --dev --opensearch.hosts=$OPENSEARCH_HOSTS --opensearch.ignoreVersionMismatch=true --server.host=$SERVER_HOST",
+    "start:security": "scripts/use_node scripts/opensearch_dashboards --dev --security",
+    "start:security:default": "export OPENSEARCH_USERNAME=kibanaserver && export OPENSEARCH_PASSWORD=kibanaserver && scripts/use_node scripts/opensearch_dashboards --dev --security",
```

Comment:
I’m not comfortable with this. Where is this default defined? Is it something we use in CI or is it a default for OS?

Reply:
Yeah I was uncomfortable with this. It is the demo user for OpenSearch Dashboards defined by the OpenSearch Security plugin:

https://github.com/opensearch-project/security/blob/main/config/internal_users.yml#L27